pkg/fsp/header_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017-2018 the LinuxBoot Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package fsp
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+)
+
+const (
+	// From https://github.com/IntelFsp/FSP/blob/master/ApolloLakeFspBinPkg/FspBin/Fsp.fd
+	// under the BSD license.
```

Comment:
As pointed by Trammel, it's not just a BSD license, also update this comment. ie:
`under the FSP License https://github.com/IntelFsp/FSP/blob/master/FSP_License.pdf.`

Reply:
whoops - I updated the content of `test_blobs` but not this comment

Reply:
fixed